index.html

```diff
@@ -2,236 +2,338 @@
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="fourth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="fifth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-
-      <div class="sixth pos">
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-    </section>
-
-    <section class="item-3">
-      <div class="about">
-        <h2 class="aboutMe">About me</h2>
-        <p class="decription">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <a class="seeProject">Get My Resume</a>
-        <img class="bottomline" src="assets/buttomline.png" alt="border line" />
-        <div class="language">
-          <div class="triangle">
-            <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle" />
-          </div>
-          <ul class="langElement">
-            <li class="lang">Language</li>
-            <li class="javascript">Javascript</li>
-            <li class="ruby">Ruby</li>
-            <li class="html">Html</li>
-            <li class="css">Css</li>
+    <main>
+      <section class="headline">
```

Comment:
- [ ] In **Desktop-version** This background-image is not showing in this `section` also. Kindly add them
- Your website  
![yerg](https://user-images.githubusercontent.com/26767383/154798675-3984aed8-71bd-4383-8eb8-24462c3f799f.PNG)

- Original website
![bdfbf](https://user-images.githubusercontent.com/26767383/154798679-4b52e6d3-0d3d-467e-9496-e03518e5fbfa.PNG)

Reply:
chages made